modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4041,6 +4052,9 @@ A program using pyramid scaling, Canny, contours and contour simplification to f
 squares in the input image.
 */
 
+CV_EXPORTS_W void findContoursLinkRuns(InputArray image, OutputArrayOfArrays contours, OutputArray hierarchy);
+CV_EXPORTS_W void findContoursLinkRuns(InputArray image, OutputArrayOfArrays contours);
```

Comment:
Could you add documentation for the functions? Do we really need it in public API?

Reply:
It was separate mode in `findContours` - CV_LINK_RUNS, but it is very different from other modes (Suzuki algorithm), for example it doesn't require internal image copying and threshold. It also supports only RETR_LIST results output. So I thought that it would be better to extract it to separate function. I don't know if it is useful for anyone or have any bugs, because it was not tested or called anywhere in samples. I've added result verification by comparing it with old implementation output in the `link_runs` test, but I haven't checked it beyond that.

I also can not find paper describing this algorithm, maybe @vpisarev could help?

I can remove this function if we decide we don't need it.